pkg/distlock/locker.go

```diff
@@ -19,3 +19,26 @@ type Locker interface {
 	// Unlock should return the distributed lock to the implemented synchronization point.
 	Unlock() error
 }
+
+type LockerMock struct{}
```

Comment:
This will be used in Clair, when Clairctl utilizes a jsonblob store.

Reply:
Does this need to be here? clairctl can just provide a bogus implementation.

Reply:
fair, I'll gut it.